easybuild/easyconfigs/c/cwltool/cwltool-3.1.20221018083734-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,108 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'cwltool'
+version = '3.1.20221018083734'
+
+homepage = 'https://github.com/common-workflow-language/cwltool'
+description = """Common workflow language (CWL) reference implementation."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
Since you require `networkx`, use `foss`as toolchain (and remove the `foss` part for `networkx` below)

Reply:
Done